samples/bookinfo/routing-rules/route-rule-all-v1.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: productpage-route
+spec:
+  hosts:
+  - productpage
+  gateways:
+  - bookinfo-gateway
+  - mesh #TODO remove this if not needed
+  http:
+  - route:
+    - destination:
+        name: productpage
+        subset: v1
+      weight: 100
```

Comment:
I think weight is optional if its 100.

Reply:
yes it is optional but people seem to like to see it explicitly so I added it in the examples. That said, I'm also OK with leaving it out if you feel strongly about it. It is a lot of unnecessary noise.

Reply:
I removed the unnecessary weight fields.